src/Components/Sidebar/Sidebar.jsx

```diff
@@ -146,14 +175,22 @@ export const Sidebar = () => {
               })}
             </ul>
           )}
-          <Link to="/archives" element={<ArchivedPage />}>
+          <Link to="/archives">
             <li
               className={`${location.pathname === "/archives" && "selected"}`}
```

Comment:
For navigation link router-dom provides different component
`<NavLink className={(navigationData) => navigationData.isActive ? styles.active : null} to="/explore"> </NavLink>`